pkgs/flatpak/com.github.rafostar.Clapper.yml

```diff
@@ -11,13 +11,34 @@ finish-args:
   - --filesystem=xdg-videos
   - --socket=pulseaudio
   - --share=network
+  - --env=GST_PLUGIN_PATH=/app/lib/codecs/lib/gstreamer-1.0
 
 # maybe add additinal codecs?
 
 modules:
+  # gstreamer from the gnome runtime is outdated, maybe upgrade all modules?
```

Comment:
> gstreamer from the gnome runtime is outdated, maybe upgrade all modules?

Its outdated in 3.36 runtime. The 3.38 will have the latest version. We always have to make sure that modules we include match the gstreamer minor version from runtime. Otherwise it wont work. There is probably no need to compile more modules then necessary, as each time a new GNOME version is released the latest GStreamer appears in the runtime shortly afterwards.

Reply:
i know, I just didn't know if you depend on anything > 1.16.2. I used to correct gst-vaapi version

Reply:
The gstreamer 1.18 was just released few days ago. As of now all code was made using 1.16. I do not know if 3.38 + 1.18 runtime is already available. We can always update the `.yml` file later.